container/incremental_load.sh.tpl

```diff
@@ -277,6 +277,18 @@ if [[ "%{run}" == "True" ]]; then
   # We can clean up before we exec docker, since the exit handler will no longer run.
   cleanup
 
+  # Bash treats empty arrays as unset variables for the purposes of `set -u` - work around this.
```

Comment:
nit: to be clear, `set -u` is not an issue but a kinda runtime linter. So this is not a workaround but a bug fix. Could you rephrase the comment to look not like a workaround?